udata/core/dataset/models.py

```diff
@@ -181,26 +177,14 @@ def closed_format(self):
         """Return True if the specified format is in CLOSED_FORMATS."""
         return self.format.lower() in CLOSED_FORMATS
 
-    def check_availability(self, group):
-        """Check if a resource is reachable against a Croquemort server.
-
-        Return a boolean.
-        """
-        if self.filetype == 'remote':
-            # We perform a quick check for performances matters.
-            error, response = check_url_from_cache(self.url, group)
-            if error or 'status' not in response:
-                return False
-            elif int(response['status']) >= httplib.INTERNAL_SERVER_ERROR:
-                return False
-            else:
-                return True
-        else:
-            return True  # We consider that API cases (types) are OK.
+    def check_availability(self):
+        '''
+        Return the check status from extras if any.
 
-    @property
-    def is_available(self):
-        return self.check_availability(group=None)
+        NB: `unknown` will evaluate to True in the aggregate checks using
```

Comment:
Tricky

Reply:
It kind of is :-/ I hesitated with `None` but the advantage of this solution is that we don't need to modify the aggregate checks (`all([])`).